tests/block/k8s/create_block_test.go

```diff
@@ -57,7 +60,12 @@ func (s *K8sBlockImageCreateSuite) SetupSuite() {
 
 	var err error
 
-	s.testClient, err = clients.CreateTestClient(tests.Platform)
+	rook, err := installer.NewK8sRookhelper()
```

Comment:
could you just call `NewK8sRookHelper` inside the `InstallRookOnK8s`? Seems like you're not using that `rook` context anywhere after this